### ISSUE TITLE
Use aggregate layer model when a layer has more than one model

### ISSUE THREAD

**Issue by loicgasser** [geoadmin/mf-chsdi3#1248]
This requires a huge refactoring.
@procrastinatio 
We can discuss how we want to implement that here.

**Comment by procrastinatio** [comment]
Yes, but what do you mean by aggregate ? On the client-side, aggreagate only means many layers used at different scale/resolution.

**Comment by loicgasser** [comment]
I quote you:

I think, for now, the diffent models are used at different scale, so only the model valid at a specific scale should be used and its features returned. Why don't we used an ggregate layer in this case ? Or just simply swap the geomtry column ?

**Comment by daguer** [comment]
For info, this layer has this problem:
ch.bfe.windenergieanlagen

(in PROD at the next deploy 20150415)

**Comment by procrastinatio** [comment]
No user value and much more important tasks for improving data management (or at least trying).
